bug: Cannot find module '@ionic/angular-server' or its corresponding type declarations

### Prerequisites

- [X] I have read the [Contributing Guidelines](https://github.com/ionic-team/ionic-framework/blob/main/.github/CONTRIBUTING.md#creating-an-issue).
- [X] I agree to follow the [Code of Conduct](https://ionicframework.com/code-of-conduct).
- [X] I have searched for [existing issues](https://github.com/ionic-team/ionic-framework/issues) that already report this problem, without success.

### Ionic Framework Version

- [ ] v4.x
- [ ] v5.x
- [X] v6.x

### Current Behavior

Error: src/app/app.server.module.ts:6:35 - error TS2307: Cannot find module '@ionic/angular-server' or its corresponding type declarations.

6 import { IonicServerModule } from '@ionic/angular-server';
                                    ~~~~~~~~~~~~~~~~~~~~~~~


Error: src/app/app.server.module.ts:9:12 - error NG1010: Value at position 2 in the NgModule.imports of AppServerModule is not a reference Value could not be determined statically.

### Expected Behavior

Angular SSR 

### Steps to Reproduce

npm run dev:ssr

### Code Reproduction URL

_No response_

### Ionic Info

_No response_

### Additional Information

Angular 13

--- a/packages/angular-server/package.json
+++ b/packages/angular-server/package.json
@@ -19,6 +19,9 @@
   "bugs": {
     "url": "https://github.com/ionic-team/ionic/issues"
   },
+  "publishConfig": {
+    "directory": "dist"
+  },
   "homepage": "https://ionicframework.com/",
   "scripts": {
     "prepublishOnly": "npm run build.prod",

--- a/packages/angular-server/tsconfig.json
+++ b/packages/angular-server/tsconfig.json
@@ -2,7 +2,7 @@
   "extends": "../../tsconfig",
   "compilerOptions": {
     "baseUrl": "./packages/angular-server",
-    "module": "commonjs",
+    "module": "es2015",
     "rootDir": "src",
     "outDir": "dist"
   },

